src/v2/Apps/Conversation/Components/OrderUpdate.tsx

```diff
@@ -15,12 +16,17 @@ import { TimeSince } from "./TimeSince"
 import { OrderUpdate_event } from "../../../__generated__/OrderUpdate_event.graphql"
 export interface OrderUpdateProps {
   event: OrderUpdate_event
+  setShowDetails: (showDetails: boolean) => void
 }
 
-export const OrderUpdate: React.FC<OrderUpdateProps> = ({ event }) => {
+export const OrderUpdate: React.FC<OrderUpdateProps> = ({
+  event,
+  setShowDetails,
```

Comment:
IMO this level of prop drilling makes the maintenance complicated, when we have to pass it down to the grandchildren I think we should provide a more elegant solution, like a shared hook among them, context API, store, etc.

Reply:
I know this prop drilling was here before, so I don't wanna block the ticket, just flagging bc I've seen so many components we have to touch in order to call a common callback among conversation components.

@jo-rs can we have a refactor ticket for it? 💟 

another good example(from the pov that needs to refactor) is this [guy](https://github.com/artsy/force/blob/dd1088ea9a638fa5fd2e24bb7c2e4ec1b606a27b/src/v2/Apps/Conversation/Components/Conversation.tsx#L33) ideally we would just have something like a callback and the pagination/refetch container would handle when we load more, instead we're prop drilling `relay.environment` when we have hooks that you can use to import it, just closing pointing out that we need a smart way to handle shared callbacks and states(rn not needed) in the conversation context instead of keeping this prop drilling here.

Reply:
Oh, sorry, I missed these messages last week 😕 
Agreed @araujobarret , a hook or context might make sense here. I'll create a ticket for a refactor. Thank you! 🙏